<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JanelaPrincipal</class>
 <widget class="QWidget" name="JanelaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PaintOpenGL</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QMenuBar" name="menuBar">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>25</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="defaultUp">
      <bool>false</bool>
     </property>
     <property name="nativeMenuBar">
      <bool>true</bool>
     </property>
     <widget class="QMenu" name="menuArquivo">
      <property name="geometry">
       <rect>
        <x>367</x>
        <y>135</y>
        <width>172</width>
        <height>140</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tearOffEnabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Arquivo</string>
      </property>
      <property name="separatorsCollapsible">
       <bool>true</bool>
      </property>
      <addaction name="actionNovo"/>
      <addaction name="openAction"/>
      <addaction name="saveAction"/>
      <addaction name="separator"/>
      <addaction name="exitAction"/>
     </widget>
     <addaction name="menuArquivo"/>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolBar" name="mainToolBar">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <addaction name="actionNovo"/>
     <addaction name="openAction"/>
     <addaction name="saveAction"/>
     <addaction name="exitAction"/>
    </widget>
   </item>
   <item row="2" column="0" rowspan="10" colspan="5">
    <widget class="PainelOpenGl" name="painelGL" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="5" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="8" column="0">
      <widget class="QPushButton" name="reflexXBtn">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Ref X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="widthBtn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resourcesfiles.qrc">
         <normaloff>:/icons/icons/View-Width-icon.png</normaloff>:/icons/icons/View-Width-icon.png</iconset>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>Lados:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="shXDoubleSpinBox">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="minimum">
        <double>-99.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="reflexYBtn">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Ref Y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="yScaleSpinBox">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="minimum">
        <double>-9999000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>SH X</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="meshCheckBox">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Malha</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QCheckBox" name="proportionalScaleCb">
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Proporcional</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="xScaleSpinBox">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>-99990000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBox">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="minimum">
        <number>3</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="heightBtn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resourcesfiles.qrc">
         <normaloff>:/icons/icons/View-Height-icon.png</normaloff>:/icons/icons/View-Height-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="shYDoubleSpinBox">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="minimum">
        <double>-99.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>SH Y</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDoubleSpinBox" name="meshSizeSpin">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="6">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>219</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="6">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>CÃ¢mera</string>
     </property>
    </widget>
   </item>
   <item row="5" column="6">
    <widget class="QPushButton" name="upBtn_2">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resourcesfiles.qrc">
       <normaloff>:/icons/icons/u_arrow_icon.png</normaloff>:/icons/icons/u_arrow_icon.png</iconset>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QPushButton" name="leftBtn_2">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resourcesfiles.qrc">
       <normaloff>:/icons/icons/l_arrow_icon.png</normaloff>:/icons/icons/l_arrow_icon.png</iconset>
     </property>
    </widget>
   </item>
   <item row="6" column="6">
    <widget class="QPushButton" name="downBtn_2">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resourcesfiles.qrc">
       <normaloff>:/icons/icons/d_arrow_icon.png</normaloff>:/icons/icons/d_arrow_icon.png</iconset>
     </property>
    </widget>
   </item>
   <item row="6" column="7">
    <widget class="QPushButton" name="rightBtn_2">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/r_arrow_icon.png</normaloff>../icons/r_arrow_icon.png</iconset>
     </property>
    </widget>
   </item>
   <item row="7" column="5" colspan="2">
    <widget class="QCheckBox" name="checkBoxIluminacao">
     <property name="text">
      <string>IluminaÃ§Ã£o</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="5" colspan="2">
    <widget class="QPushButton" name="upCamBt">
     <property name="text">
      <string>3D</string>
     </property>
    </widget>
   </item>
   <item row="9" column="5" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>2D</string>
     </property>
    </widget>
   </item>
   <item row="10" column="5" colspan="3">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QDial" name="rotationDial">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>64</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>ClosedHandCursor</cursorShape>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="6" rowspan="2">
    <widget class="QPushButton" name="upBtn">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resourcesfiles.qrc">
       <normaloff>:/icons/icons/u_arrow_icon.png</normaloff>:/icons/icons/u_arrow_icon.png</iconset>
     </property>
    </widget>
   </item>
   <item row="12" column="0" rowspan="2">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="backgroundObjectLabel">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Fundo:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="colorBtn">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lineObjectLabel">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Linha:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="lineColorBtn">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="12" column="1" rowspan="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QPushButton" name="drawCircleBtn">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resourcesfiles.qrc">
         <normaloff>:/icons/icons/perfect-circle_icon-icons.com_53928.png</normaloff>:/icons/icons/perfect-circle_icon-icons.com_53928.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="drawSquareBtn">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resourcesfiles.qrc">
         <normaloff>:/icons/icons/squareroundedemptyoutlinedbuttonshape_80680.png</normaloff>:/icons/icons/squareroundedemptyoutlinedbuttonshape_80680.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="drawBtn">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resourcesfiles.qrc">
         <normaloff>:/icons/icons/67351.png</normaloff>:/icons/icons/67351.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="eraseBtn">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resourcesfiles.qrc">
         <normaloff>:/icons/icons/erase.png</normaloff>:/icons/icons/erase.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="12" column="2" rowspan="2">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="1">
      <widget class="QPushButton" name="zoomOutBtn">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resourcesfiles.qrc">
         <normaloff>:/icons/icons/54349.png</normaloff>:/icons/icons/54349.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="zoomInBtn">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resourcesfiles.qrc">
         <normaloff>:/icons/icons/23910.png</normaloff>:/icons/icons/23910.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="12" column="3" rowspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="focusPreviousBtn">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resourcesfiles.qrc">
         <normaloff>:/icons/icons/ic_skip_previous_black_36dp.png</normaloff>:/icons/icons/ic_skip_previous_black_36dp.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="focusNextBtn">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resourcesfiles.qrc">
         <normaloff>:/icons/icons/ic_skip_next_black_36dp.png</normaloff>:/icons/icons/ic_skip_next_black_36dp.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="12" column="4">
    <widget class="QLabel" name="objectNumberLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="13" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>468</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="5">
    <widget class="QPushButton" name="leftBtn">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resourcesfiles.qrc">
       <normaloff>:/icons/icons/l_arrow_icon.png</normaloff>:/icons/icons/l_arrow_icon.png</iconset>
     </property>
    </widget>
   </item>
   <item row="13" column="6">
    <widget class="QPushButton" name="downBtn">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resourcesfiles.qrc">
       <normaloff>:/icons/icons/d_arrow_icon.png</normaloff>:/icons/icons/d_arrow_icon.png</iconset>
     </property>
    </widget>
   </item>
   <item row="13" column="7">
    <widget class="QPushButton" name="rightBtn">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/r_arrow_icon.png</normaloff>../icons/r_arrow_icon.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionNovo">
   <property name="icon">
    <iconset resource="resourcesfiles.qrc">
     <normaloff>:/icons/icons/ic_insert_drive_file_black_24dp.png</normaloff>:/icons/icons/ic_insert_drive_file_black_24dp.png</iconset>
   </property>
   <property name="text">
    <string>Novo</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="resourcesfiles.qrc">
     <normaloff>:/icons/icons/ic_vertical_align_bottom_black_24dp.png</normaloff>:/icons/icons/ic_vertical_align_bottom_black_24dp.png</iconset>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="icon">
    <iconset resource="resourcesfiles.qrc">
     <normaloff>:/icons/icons/ic_exit_to_app_black_24dp.png</normaloff>:/icons/icons/ic_exit_to_app_black_24dp.png</iconset>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset resource="resourcesfiles.qrc">
     <normaloff>:/icons/icons/ic_folder_black_24dp.png</normaloff>:/icons/icons/ic_folder_black_24dp.png</iconset>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PainelOpenGl</class>
   <extends>QWidget</extends>
   <header location="global">painelopengl.h</header>
   <container>1</container>
   <slots>
    <signal>mouseEvent()</signal>
    <slot>alterarRaio(double)</slot>
    <slot>alterarLados(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resourcesfiles.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exitAction</sender>
   <signal>triggered()</signal>
   <receiver>JanelaPrincipal</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
